src/UNL/UCBCN/Manager/ArchiveCalendar.php

```diff
@@ -7,7 +7,7 @@
 use UNL\UCBCN\Permissions;
 use UNL\UCBCN\Manager\Controller;
 
-class ArchiveCalendar extends PostHandler
```

Comment:
This request should still be a POST because data is being altered

Reply:
I guess I'm going through commits one at a time and there are more recent changes, nm